pkg/mailer/template.go

```diff
@@ -130,7 +130,7 @@ Please consider to clean up the project directory (i.e. /project/{{.ProjectID}}
 
 If more quota is needed, please see the procedure described in the "Exceptional quota requests" section of the following intranet page: https://intranet.donders.ru.nl/index.php?id=quota
 
-If you have further questions, don't hesitate to contact the TG helpdesk <helpdesk@fcdonders.ru.nl>.
+If you have further questions, don't hesitate to contact the TG helpdesk <helpdesk@donders.ru.nl>.
```

Comment:
I guess the same goes for the helpdesk email address

Reply:
@linusband This is a hot fix and should be done as quick as possible.  We shouldn't point end user to the out-dated SOP document.

I have created a new issue #19 to factor out the email template in general.  For not, could you accept this change to get it fixed as quick as possible?

Actually if we categorize it as an "emergency" change, we are allowed to register it (through the review process) after it has been fixed.